worldedit-core/src/main/java/com/sk89q/worldedit/extent/reorder/MultiStageReorder.java

```diff
@@ -42,7 +42,10 @@
 
 /**
  * Re-orders blocks into several stages.
+ *
+ * @deprecated Use {@link com.sk89q.worldedit.extent.world.SideEffectExtent} with post edit instead.
```

Comment:
`post edit` typo? Not sure what you wanted to refer to here. Maybe link to `#setPostEditSimulationEnabled`?